<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MWindowClass</class>
 <widget class="QWidget" name="MWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">MWindowCLass{background-image:url(:/image/bk3.jpg);}</string>
  </property>
  <widget class="QRadioButton" name="Rdb_prices">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>270</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>花费最少</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Rdb_Time">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>270</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>距离最短</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Rdb_Change">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>270</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>转机次数最少</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Cbb_Start">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="Cbb_End">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>171</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>始发地</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>目的地</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PsBt_Booked">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>0</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>已定机票</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PsBt_Search">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>320</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
